source/code/source/playfab/PlayFabCurlHttpPlugin.cpp

```diff
@@ -133,11 +139,15 @@ namespace PlayFab
             return nitems * size; // The return expected by curl for this callback
         }
 
+size_t headerKeyLength = size;
+#if defined(PLAYFAB_PLATFORM_GDK)
+        headerKeyLength = nitems;
+#endif
         // If this header-line is long enough, and the header starts with the key we expect
-        if ((nitems > requestIdheaderKeyLen) && (strncasecmp(buffer, requestIdHeaderKey, requestIdheaderKeyLen) == 0))
+        if ((headerKeyLength > requestIdheaderKeyLen) && (strncasecmp(buffer, requestIdHeaderKey, requestIdheaderKeyLen) == 0))
```

Comment:
I'm a little worried this is a change in behavior for non-GDK. Previously, for non-GDK, we would compare nitems to requestIdheaderKeyLen. Now, though, for non-GDK, we effectively compare size to requestIdheaderKeyLen. Does the #if above need to be an #ifndef

Reply:
I'm seeing the error that caused this swap is no longer triggering, I'll remove the change.